docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,225 @@
+
+1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
+1. Choose from the following, as needed:
+    * **Collection**. Manage [Collectors and Sources](/docs/send-data/collection), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector), [Data Collection Status](/docs/manage/ingestion-volume/collection-status-page), [Ingest Budget](/docs/manage/ingestion-volume/ingest-budgets), Archive, and [Data Archiving](/docs/manage/data-archiving).
+    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/search/search-query-language/parse-operators), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).
+    * **Metrics**. Manage [metrics rules](/docs/metrics/metric-rules-editor), [logs-to-metrics](../metrics/logs-to-metrics.md), and [metrics transformation rules](../metrics/metrics-transformation-rules.md).
+    * **Cloud SIEM Integrations**. Manage [Ingest Mappings](/docs/cse/ingestion/sumo-logic-ingest-mapping), [Log Mappings](/docs/cse/schema/create-structured-log-mapping), [Context Actions](/docs/cse/administration/create-cse-context-actions), [Actions](/docs/cse/administration/create-cse-actions), [Enrichment](/docs/cse/integrations/enrichments-and-indicators), [Automation](/docs/cse/automation).
+    * **Cloud SIEM Entities**. Manage [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group), [Normalization](/docs/cse/records-signals-entities-insights/configure-entity-lookup-table/), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type), and [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality).
+    * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).
+    * **Cloud SOAR Configurations**. Manage [Incidents](/docs/cloud-soar/incidents-triage/), [Fields](/docs/cloud-soar/incidents-triage/#custom-fields), and [Incident Labels](/docs/cloud-soar/global-functions-menu/#incident-label).
+
+### Manage accounts, users, and security
+
+Sumo Logic administrators (admins) manage user accounts, user roles, and security. You must have Sumo Logic Admin role privileges to perform these tasks.
+
+To administer Sumo Logic accounts, users, and security, do the following:
+
+1. From the top navigation bar, click **Administration**.<br/><img src={useBaseUrl('img/get-started/admin.png')} alt="img/get-started/admin.png" width="400"/>
+1. Choose from any of the following:
```

Comment:
Like in my previous comment, for the items in this list, should you show them exactly as they appear in the UI?

Reply:
fixed!